zimui/src/views/VideoPlayerView.vue

```diff
@@ -75,6 +75,10 @@ const videoURL = computed<string>(() => {
   return video.value?.videoPath || ''
 })
 
+const videoFormat = computed<string>(() => {
```

Comment:
Please add comment explaining this is a kind of trick to retrieve video format and add one or two sample values (I pretty sure it will always work, but should it become a problem it needs to be clear what this is doing)

Reply:
Added in [832ac24](https://github.com/openzim/youtube/pull/259/commits/832ac248db071e4a381764721beb9b8049c0786c)